### PR TITLE
Add shape drawing order to server API

### PR DESCRIPTION
**Pull request by danielrwolff**
you asked. probably doesn't compile.

### CODE REVIEW

File: srv/server.py [hunk 1]
```diff
@@ -69,14 +71,13 @@ async def get_shapes(beg_time: int = None, end_time: int = None) -> Shapes:
...
                 }
             )
 
-        resp = Shapes(
-            circles=circle_resp,
-            rectangles=rect_resp,
-        )
-
+        resp = circle_resp + rect_resp
+        resp.sort(lambda x: x.created_at)
```
Comment (Mandrenkov) [inline srv/server.py]:
```suggestion
        resp.sort(lambda shape: shape["created_at"])
```

File: srv/server.py [hunk 2]
```diff
@@ -58,6 +58,8 @@ async def get_shapes(beg_time: int = None, end_time: int = None) -> Shapes:
                     "x": circle[2],
                     "y": circle[3],
                     "r": circle[4],
+                    "created_at": circle[5],
```
Comment (Mandrenkov) [inline srv/server.py]: I think we'll need to convert the `created_at` fields into strings or integers before serializing it to JSON.

File: srv/server.py [hunk 3]
```diff
@@ -69,14 +71,13 @@ async def get_shapes(beg_time: int = None, end_time: int = None) -> Shapes:
...
+                    "type": "rect",
                 }
             )
 
-        resp = Shapes(
-            circles=circle_resp,
-            rectangles=rect_resp,
-        )
-
+        resp = circle_resp + rect_resp
```
Comment (Mandrenkov) [inline srv/server.py]: We'll need to change the `Shapes` class as well to match the new format (see `response_model` at the top of the endpoint).

**Review by Mandrenkov — CHANGES_REQUESTED** [review]

**Comment by Mandrenkov** [comment]
Thanks @danielrwolff!

File: srv/server.py (hunk 1 above)
Comment (danielrwolff) [reply srv/server.py]: Done

File: srv/server.py (hunk 2 above)
Comment (danielrwolff) [reply srv/server.py]: Trying `int(circle[5].timestamp())` but I don't know how to run the code so no idea if this works.

File: srv/server.py (hunk 3 above)
Comment (danielrwolff) [reply srv/server.py]: Replacing `Shapes` class with `list[Shape]` - a list of a new object that both `Circle` and `Rectangle` extend.

**Comment by JacobyJoukema** [comment]
LGTM

**Review by JacobyJoukema — APPROVED** [review]
😯 LGTM 🔥🔥🔥🚀🚀🚀

**Review by Mandrenkov — APPROVED** [review]
Not sure if this works but 🤷‍♂️.

**Pull request merged**